interface-definitions/protocols-pim.xml.in

```diff
@@ -41,8 +57,134 @@
                   </constraint>
                 </properties>
               </leafNode>
+              <node name="igmp">
+                <properties>
+                  <help>Internet Group Management Protocol (IGMP) options</help>
+                </properties>
+                <children>
+                  <leafNode name="version">
+                    <properties>
+                      <help>IGMP version used on this interface</help>
```

Comment:
Tiny suggestion: maybe just "for this interface" for a less wordy help string?

Reply:
"Interface IGMP version" even shorter?